src/components/App/index.js

```diff
@@ -23,7 +20,7 @@ const Main = () => (
 );
 
 
-const handleSelect = (selectedKey) => {
+const _handleSelect = (selectedKey) => {
```

Comment:
I think you should undo this, then change the reference below from 'this.handleSelect' to 'handleSelect'. That should solve the issue. If not, another thing to look at together.